internal/handler/wallet/backup_test.go

```diff
@@ -26,24 +29,110 @@ func Test_walletBackupAccount_Handle(t *testing.T) {
 
 	nickname := "walletToBackup"
 	password := "zePassword"
-	_, errGenerate := wallet.Generate(nickname, password)
-	assert.Nil(t, errGenerate)
+	wlt, walletError := wallet.Generate(nickname, password)
+	assert.Nil(t, walletError)
+
+	walletError = wlt.Unprotect(password)
+	assert.Nil(t, walletError)
 
 	t.Run("invalid nickname", func(t *testing.T) {
 		resp := backupWallet(t, api, "toto")
 		verifyStatusCode(t, resp, http.StatusNotFound)
 	})
 
-	t.Run("export success", func(t *testing.T) {
+	t.Run("invalid backup method", func(t *testing.T) {
+		testResult := make(chan walletapp.EventData)
+
+		go func() {
+			prompterApp.App().PromptInput <- "unknown method"
+			testResult <- (<-resChan)
+		}()
+
+		resp := backupWallet(t, api, nickname)
+		verifyStatusCode(t, resp, http.StatusUnauthorized)
```

Comment:
[optional]

here it should be status bad request

optional because it's not the first time we make this mistake